test/e2e/frictionless/step-definitions/dc.steps.js

```diff
@@ -400,3 +402,32 @@ Then(/^I read expected analytics data with replacements "([^"]*)"$/, async funct
 
   this.page.wikiAnalyticsData = events;
 });
+
```

Comment:
Can we move the CaaS steps to https://git.corp.adobe.com/wcms/Platform-UI-Adobe?

https://www.npmjs.com/package/@amwp/platform-ui-lib-adobe

Other teams can use these steps too.

Reply:
If I understood correctly, you want me to port this changes to https://git.corp.adobe.com/wcms/Platform-UI-Adobe, raise the PR there and close this one ?

Reply:
It's fine to keep the CaaS steps in this repo for now. We can move them when other teams want to use. In the meantime, we may have to stabilize those steps.